src/components/CardArt.vue

```diff
@@ -40,13 +40,24 @@ const showErrorOverlay = computed(() => {
 <template>
   <div class="art-frame">
     <div v-if="showErrorOverlay" class="overlay error">
-      <div class="shape-x"></div>
+      <div>
+        <div class="shape-x"></div>
+        <p>There was an error loading the card art.</p>
+      </div>
     </div>
     <div v-else-if="showLoadingOverlay" class="overlay loading">
       <LoadingPulser class="pulser" />
+      <p class="vh">Loading next card art...</p>
     </div>
     <div class="flavor-mask" v-if="card.flavor_name">Hint: this print has a flavor name.</div>
-    <img alt="" class="vh preload" :src="imageUri" :onload="onLoad" :onerror="onError" />
+    <img
+      alt=""
+      class="vh preload"
+      :src="imageUri"
+      :onload="onLoad"
+      :onerror="onError"
+      data-testid="card-art-preload"
```

Comment:
needed to add a data-testid here to target it in the test to trigger the load and error events.